tests/providers/amazon/aws/operators/test_glue.py

```diff
@@ -38,10 +37,7 @@
 
 
 class TestGlueJobOperator:
-    @pytest.fixture(autouse=True)
-    def setup_method(self):
```

Comment:
Here and in the other places - running "load_test_config" is not needed - "load_test_config" is run automatically via pytest fixture, and there is no need to run them again here.